holoviews/plotting/bokeh/util.py

```diff
@@ -125,7 +128,7 @@ def mpl_to_bokeh(properties):
     return new_properties
 
 
-def layout_padding(plots):
+def layout_padding(plots, renderer):
     """
     Temporary workaround to allow empty plots in a
     row of a bokeh GridPlot type. Should be removed
```

Comment:
Yes, will update the docstring (and perhaps reopen the issue). It's not actually fixed.